common/djangoapps/student/admin.py

```diff
@@ -166,6 +166,16 @@ class UserAdmin(BaseUserAdmin):
     """ Admin interface for the User model. """
     inlines = (UserProfileInline,)
 
+    def get_readonly_fields(self, request, obj=None):
+        """
+        Allows editing the users while skipping the username check, so we can have Unicode username with no problems.
```

Comment:
Please, separate title from the body of the docstring.

Reply:
Done